libs/ledger-live-common/src/families/bitcoin/wallet-btc/__tests__/wallet.explorer.unit.test.ts

```diff
@@ -0,0 +1,195 @@
+                value: "30000",
+                address:
+                  "bc1pd85hsqr75rg6fwwd8kmftd3tv26tgp3spkfmn4uga6t5xsv2talqmzqt0h",
+                script_signature: "",
+                txinwitness: [
+                  "ca817c52fdb82530b15e3f35ca336257f69f649ca1b987aefe445d9ec2004e0dd475725b3d1c3496a5d82c4cc3af0119f94e3400d165c271b60865a05faea1e0",
+                  "20117f692257b2331233b5705ce9c682be8719ff1b2b64cbca290bd6faeb54423eac0619f5aee08701750063036f7264010118746578742f706c61696e3b636861727365743d7574662d3800357b2270223a226272632d3230222c226f70223a226d696e74222c227469636b223a2276646278222c22616d74223a2231303030227d68",
+                  "c1117f692257b2331233b5705ce9c682be8719ff1b2b64cbca290bd6faeb54423e",
+                ],
+                sequence: 4294967293,
+              },
+            ],
+            outputs: [
+              {
+                output_index: 0,
+                value: "546",
+                address:
+                  "bc1p4hdgsumm299ncy53g5xxlexsex0qqj0sl0flr5hp7d5q6xskw7rqq3srjc",
+                script_hex:
+                  "5120adda88737b514b3c1291450c6fe4d0c99e0049f0fbd3f1d2e1f3680d1a167786",
+              },
+              {
+                output_index: 1,
+                value: "4246",
+                address: "bc1qrgt98epetd665p3the4qtsd7m30jdrte5g20k0",
+                script_hex: "00141a1653e4395b75aa062bbe6a05c1bedc5f268d79",
+              },
+            ],
+            block: {
+              hash: "0000000000000000000059c005bc8689ef7c4935034c201f94a0ef54d60b8aac",
+              height: 788060,
+              time: "2023-05-03T08:17:57Z",
+            },
+            confirmations: 920,
+          },
+        ],
+        token: null,
+      },
+    });
+
+    const params = {
+      batch_size: 10000,
+    };
+    const txs = await explorer.fetchTxs(
+      {
+        account: 0,
+        index: 0,
+        address:
+          "bc1pd85hsqr75rg6fwwd8kmftd3tv26tgp3spkfmn4uga6t5xsv2talqmzqt0h",
+      },
+      params
+    );
+    expect(txs.length).toEqual(2);
+    expect(txs[0].inputs.length).toEqual(1);
+    expect(txs[0].outputs.length).toEqual(2);
+  });
+
+  it("getRelayFee unit test", async () => {
+    // @ts-expect-error method is mocked
+    network.mockResolvedValue({
+      data: {
+        relay_fee: "0.00001000",
+        incremental_fee: "0.00001000",
+        version: "210100",
+        subversion: "/Satoshi:0.21.1/",
+      },
+    });
+    const relayFee = await explorer.getRelayFee();
+    expect(relayFee).toBeGreaterThan(0);
+  });
+
+  it("getFees unit test", async () => {
+    // @ts-expect-error method is mocked
+    network.mockResolvedValue({
+      data: { "2": 555985, "4": 504992, "6": 479059, last_updated: 1683675584 },
+    });
+    const fees = await explorer.getFees();
+    expect(Object.keys(fees).length).toBeGreaterThan(3);
+    expect(fees["2"]).toEqual(555985);
+  });
+
+  it("getCurrentBlock unit test", async () => {
```

Comment:
As a overall pattern, i think we should write "describe(methodName)", and then represent the specs a indivuals "it" instances, example :

describe("getCurrentBlock")
it("should return block height and hash using currentBlock endpoint")
it("should return null if currentBlock endpoint returns an error")

I didn't check the currentBlock method but you get the idea

Reply:
fixed